### PR TITLE
New readme

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
new readme and repl. DON'T MERGE THIS YET. repl is not in production but any day now...

@jmburges @aviflombaum

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,211 @@
...
+> Hashes are a lot like arrays: they have a bunch of slots that can point to various objects. However, in an array, the slots are lined up in a row, and each one is numbered (starting from zero). In a hash, the slots aren’t in a row (they are just sort of jumbled together) and you can use any object to refer to a slot, not just a number. 
+> 
+> –– [Chris Pine's Learn to Program, page 108](http://books.flatironschool.com/books/43?page=108)
+
+
+Up until this point, we've stored our data in list-form using arrays. Let's take a look at an example where an array would not be sufficient to meet our storage needs.
+
+### Example: Customs Inspector
+
+![Herman Melville](https://upload.wikimedia.org/wikipedia/commons/thumb/f/f1/Herman_Melville.jpg/160px-Herman_Melville.jpg)
```
Comment (jmburges) [inline README.md]: DL this and add it to a our S3. Don't want to steal bandwidth from wikimedia and want to protect against it going away

Comment (SophieDeBenedetto) [inline README.md]: done.

On Tue, Jul 7, 2015 at 3:19 PM, Joe Burgess notifications@github.com
wrote:

> In README.md
> https://github.com/learn-co-curriculum/hash-overview-readme/pull/1#discussion_r34079121
> :
> 
> >  ## Introduction
> > -\* "Hashes are a lot like arrays: they have a bunch of slots that can point to various objects. However, in an array, the slots are lined up in a row, and each one is numbered (starting from zero). In a hash, the slots aren’t in a row (they are just sort of jumbled together) and you can use any object to refer to a slot, not just a number." - From [Chris Pine's Learn to Program, page 108](http://books.flatironschool.com/books/43?page=108)
> > +> Hashes are a lot like arrays: they have a bunch of slots that can point to various objects. However, in an array, the slots are lined up in a row, and each one is numbered (starting from zero). In a hash, the slots aren’t in a row (they are just sort of jumbled together) and you can use any object to refer to a slot, not just a number.
> > +>
> > +> –– [Chris Pine's Learn to Program, page 108](http://books.flatironschool.com/books/43?page=108)
> > +
> > +
> > +Up until this point, we've stored our data in list-form using arrays. Let's take a look at an example where an array would not be sufficient to meet our storage needs.
> > +
> > +### Example: Customs Inspector
> > +
> > +Herman Melville
> 
> DL this and add it to a our S3. Don't want to steal bandwidth from
> wikimedia and want to protect against it going away
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/learn-co-curriculum/hash-overview-readme/pull/1/files#r34079121
> .

## 

Sophie DeBenedetto

File: README.md [hunk 2]
```diff
@@ -1,68 +1,211 @@
...
+
+Here's where a hash can come in handy. With a hash, we can store a list of associated pairs. In other words, we can store the pairs of item/count. Let's take a look: 
+
+```ruby
+old_fashioned_things = {
+  "whale bone corset" => 5, 
+  "porcelain vase" => 2, 
+  "oil painting" => 3
+}
+```
```
Comment (jmburges) [inline README.md]: Another great usage of hashes is named data. So something like

```
joe = {"name" => "Joe Burgess",
"height" => 74}
```

Which if we had to describe that as an array we would do it as

```
["Joe Burgess", 74]
```

and then we would have to just remember that the 0th element is the name and the 1th element is age. It's super hard to remember that.

File: README.md [hunk 3]
```diff
@@ -1,68 +1,211 @@
...
-* Hashes are structured with keys and values.
+Hashes are structured with keys and values. Each key/value pair makes up one unit in the hash. The entire collection of key/value pairs, which are comma separated, is enclosed in curly braces, `{ }`. 
 
 ```ruby
 hash = {:key => "value", :another_key => "another value"}
 ```
 
-## Creation
-* Making a new hash can be accomplished in a couple different ways.
+You can think of a hash as a dictionary. The key is the reference point that is set equal to an associated value. In fact, a dictionary makes a pretty good hash example. Let's take a look:
```
Comment (jmburges) [inline README.md:30]: Have you gone over what symbols are?

Comment (jmburges) [inline README.md:30]: Also maybe a discussion of what can be used as a key? Answer is anything, but remember keys are unique!

File: README.md (hunk 2 above)
Comment (SophieDeBenedetto) [inline README.md]: addressed w latest commit

File: README.md (hunk 3 above)
Comment (SophieDeBenedetto) [inline README.md:30]: addressed w latest commit

**Comment by jmburges** [comment]
looks good :shipit:

**Pull request merged**